src/test/java/bio/terra/workspace/db/WorkspaceDaoTest.java

```diff
@@ -150,75 +148,74 @@ void duplicateWorkspaceFails() {
   }
 
   @Nested
-  class CloudContext {
+  class TestGcpCloudContext {
```

Comment:
I think you can give it a friendly name as an a rgumnet to `@Nested` if you want